packages/cli/__tests__/zosfiles/__unit__/upload/dtu/DirToUSS.handler.unit.test.ts

```diff
@@ -124,6 +124,7 @@ describe("Upload dir-to-uss handler", () => {
             expect((Upload.dirToUSSDir as jest.Mock).mock.calls[0][UPLOAD_OPTIONS_ARG_INDEX].attributes).toBeInstanceOf(ZosFilesAttributes);
         });
 
+        // eslint-disable-next-line jest/expect-expect
```

Comment:
The disabling of lint rules in this PR should be identical to how it was done for master branch in #1099. In cases where a lint rule was disabled for an entire file, I believe it was because all or nearly all tests in the file had the same lint error.